.github/workflows/simple-tests.yml

```diff
@@ -25,13 +25,14 @@ jobs:
     # run tests in sudo for rtprio to work
     - name: Unit Test 
       run: |
-        sudo ./Validation/test.sh critical_overcurrent
-        sudo ./Validation/test.sh critical_overtemp_discharging
-        sudo ./Validation/test.sh critical_overvolt
-        sudo ./Validation/test.sh critical_undervolt
-        sudo ./Validation/test.sh overcurrent
-        sudo ./Validation/test.sh overtemp_charging
-        sudo ./Validation/test.sh overtemp_discharging
-        sudo ./Validation/test.sh overvolt
-        sudo ./Validation/test.sh undervolt
+        sudo bash Validation/test.sh BSP/Simulator/Data/ManualTests/critical_overcurrent.json
```

Comment:
Couldn't the file path be moved into the test.sh file? would make things a bit nicer to read.